src/Workspaces/MSBuild/Test/NetCoreTests.cs

```diff
@@ -94,6 +96,27 @@ public async Task TestOpenProject_NetCoreApp()
         Assert.Empty(diagnostics);
     }
 
+    [ConditionalFact(typeof(DotNetSdkMSBuildInstalled))]
+    [Trait(Traits.Feature, Traits.Features.MSBuildWorkspace)]
+    [Trait(Traits.Feature, Traits.Features.NetCore)]
+    public async Task TestOpenInMemoryProject_NetCoreApp()
+    {
+        CreateFiles(GetNetCoreAppFiles());
+
+        var projectFilePath = GetSolutionFileName("Project.csproj");
+        var content = File.ReadAllText(projectFilePath);
+        var projectDir = Path.GetDirectoryName(projectFilePath);
+
+        await using var buildHostProcessManager = new BuildHostProcessManager(ImmutableDictionary<string, string>.Empty);
+
+        var buildHost = await buildHostProcessManager.GetBuildHostWithFallbackAsync(projectFilePath, CancellationToken.None);
+        var projectFile = await buildHost.LoadProjectAsync(projectFilePath, content, LanguageNames.CSharp, CancellationToken.None);
+        var projectFileInfo = (await projectFile.GetProjectFileInfosAsync(CancellationToken.None)).Single();
+
+        Assert.Equal(Path.Combine(projectDir, "bin", "Debug", "netcoreapp3.1", "Project.dll"), projectFileInfo.OutputFilePath);
+        Assert.Null(projectFileInfo.GeneratedFilesOutputDirectory);
```

Comment:
```suggestion
```

Just delete this line -- I presume this came along from wherever you copied it, but isn't adding any value here. Checking the output path is good since that confirms it actually ran an evaluation well.

Reply:
Done